Automattic/jetpack
# Components: Fix boost score graph null-ish value handling

Fixes Automattic/boost-cloud#367

## Proposed changes:
* Handle null-ish values in tooltip
* Adjust start date in the graph

### Other information:


## Jetpack product discussion


pc9hqz-26I-p2#comment-1648

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:
* Start storybook `cd js-packages/storybook` `pnpm run storybook:dev`
* Check **JS Packages > Components > Boost Score Tooltip**
* Try with null-ish values in different properties
